documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -259,6 +259,25 @@ include::{partialsdir}/modules/all-connectors/proc-stopping-an-incremental-snaps
 
 include::{partialsdir}/modules/all-connectors/proc-stopping-an-incremental-snapshot-kafka.adoc[leveloffset=+1]
 
+ifdef::community[]
+[id="postgresql-read-only-incremental-snapshots"]
+==== Read-only incremental snapshots
+
+The PostgreSQL connector allows for running incremental snapshots with a read-only connection to the database.
```

Comment:
```suggestion
You can configure a PostgreSQL connector that has a read-only connection to the database to run incremental snapshots.
```
I'm in the process of updating the MySQL/MariaDB docs, and will apply similar wording.

Reply:
I just update it adding 'without requiring the signal data collection table.' WDYT?

Reply:
Changed it to "a signal data collection table". I think that works, si?